NG-ZORRO官网提供的Code Editor组件demo例子有问题，编译不过

### Reproduction link
[https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.ts](https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.ts)

### Steps to reproduce
无

### What is expected?
执行官网上的demo例子能正常编译通过，ng g ng-zorro-antd:code-editor-config <name>

### What is actually happening?
问题描述：
使用官网提供的”Code Editor代码编辑器”组件，ng serve命令提示如下错误Error: node_modules/ng-zorro-antd/code-editor/typings.d.ts:10:9 - error TS2687: All declarations of 'MonacoEnvironment' must have identical modifiers.
MonacoEnvironment: Environment | undefined;
问题复现：
没有增加任何自己的代码，只是完全新建了一个工程，增加了这个editor组件
1、执行官网上提供的demo例子即可复现，ng g ng-zorro-antd:code-editor-config <name>
相关版本如下“”
"ng-zorro-antd": "^14.0.0"
@angular/core": "^14.2.0",
 "monaco-editor": "^0.34.0",

| Environment | Info |
|---|---|
| ng-zorro-antd | 14.0.0 |
| Browser | chrome |

<!-- generated by ng-zorro-issue-helper. DO NOT REMOVE -->

我升级到14 遇到的是这个报错

```js

Error: node_modules/.pnpm/ng-zorro-antd@14.0.0_tsf443wkbafymssw5dt3n4vaxi/node_modules/ng-zorro-antd/graph/interface.d.ts:5:187 - error TS2307: Cannot find module 'dagre-compound' or its corresponding type declarations.

5 import { HierarchyBaseEdgeInfo, HierarchyBaseNodeInfo, HierarchyGraphDef, HierarchyGraphEdgeDef, HierarchyGraphNodeDef, HierarchyGraphNodeInfo, HierarchyGraphOption, LayoutConfig } from 'dagre-compound';

```

> 我升级到14 遇到的是这个报错
> 
> ```js
> Error: node_modules/.pnpm/ng-zorro-antd@14.0.0_tsf443wkbafymssw5dt3n4vaxi/node_modules/ng-zorro-antd/graph/interface.d.ts:5:187 - error TS2307: Cannot find module 'dagre-compound' or its corresponding type declarations.
> 
> 5 import { HierarchyBaseEdgeInfo, HierarchyBaseNodeInfo, HierarchyGraphDef, HierarchyGraphEdgeDef, HierarchyGraphNodeDef, HierarchyGraphNodeInfo, HierarchyGraphOption, LayoutConfig } from 'dagre-compound';
> ```

If you have imported `NzGraphModule` in your project, make sure you have installed `dagre-compound`

--- a/components/code-editor/typings.ts
+++ b/components/code-editor/typings.ts
@@ -10,7 +10,7 @@ import IDiffEditorConstructionOptions = editor.IDiffEditorConstructionOptions;
 
 declare global {
   interface Window {
-    MonacoEnvironment: Environment | undefined;
+    MonacoEnvironment?: Environment | undefined;
   }
 }
 

